Sources/xcodeproj/Objects/Project/PBXProject.swift

```diff
@@ -191,6 +191,44 @@ public final class PBXProject: PBXObject {
 
         return reference
     }
+    
+    
+    /// Adds a local swift package
+    ///
+    /// - Parameter path: Relative path to the swift package
+    /// - Parameter productName: The product to depend on
+    /// - Parameter target: Target to link package product to
+    /// - Parameter addFileReference: Include a file reference to the package (defaults to main group)
+    public func addLocalSwiftPackage(path: String,
```

Comment:
Nitpick: I'd turn the `path` type into `Path`.

Reply:
It's just that it should be relative, so maybe make assertion for `path.isRelative = true` and add it to the documentation? Or would it be better to accept absolute path, too and derive the relative from the project's root folder?

Reply:
That's a good point, it should be relative. If we get the absolute path, we can't turn it into a relative because the project's absolute path is only known when the project is written. I suggest two things:
- Turn it into a `Path` type and throw if the path is not relative.
- Mention this in the line that documents the argument.

What do you think?

Reply:
Like that increased safety, will do 👍